CONTRIBUTING.md

```diff
@@ -73,3 +73,9 @@ You can find usage examples of the controls available in WinUI in the Xaml Contr
 
  which can also be installed from the Windows Store:  
  https://www.microsoft.com/p/xaml-controls-gallery/9msvh128x2zt
+ 
+ ## API spec discussions
+
+Before new features will be implemented the WinUI team will do a thorough API spec discussion (API review). This can range from a single new API to an entire new control featuring dozens of new APIs. Joining such a spec discussion is a great opportunity for developers to help ensuring that new WinUI APIs will look and feel natural and are a joy to work with. In addition, spec discussions are the follow-up to feature proposals and will go into much finer details than the initial proposal. As such, taking part in these discussions gives developers the chance to be involved in the complete development process of new WinUI features - from their initial high-level inception right down to specific implementation/behavior details.
+
+These discussions will be take place in their own [repository](https://github.com/microsoft/microsoft-ui-xaml-specs). While API specs for feature proposals will typically be linked to in the specific proposal on the WinUI repository, not all active API specs will immediately be mentioned there. Thus, if you don't want to miss out on them, we recommend that you [watch the repository](https://docs.github.com/en/enterprise/2.15/user/articles/watching-and-unwatching-repositories).
```

Comment:
An API spec not (initially) mentioned by the team on the WinUI repo for example is the [Application and Windowing Spec](https://github.com/microsoft/microsoft-ui-xaml-specs/pull/71) for WinUI 3. The spec could only be discovered by a community member if they were watching the specs repo or another member of the community mentioned it to them.